server.js

```diff
@@ -1,8 +1,11 @@
 const express = require("express");
+const router = require("./controllers");
 
 const app = express();
 const PORT = process.env.PORT || 3000;
 
+app.use(router);
+
 app.listen(() => {
```

Comment:
Looks like you're missing the port:

```suggestion
app.listen(PORT, () => {
```